Doc/library/doctest.rst

```diff
@@ -277,14 +277,37 @@ Which Docstrings Are Examined?
 The module docstring, and all function, class and method docstrings are
 searched.  Objects imported into the module are not searched.
 
-In addition, if ``M.__test__`` exists and "is true", it must be a dict, and each
+In addition, there are cases when you want tests to be part of a module but not part
+of the help text, which requires that the tests should not be placed in the docstring.
+Doctest looks for a module-level variable called ``__test__`` and uses it to locate other
+tests. If ``M.__test__`` exists and is ``True``, it must be a dict, and each
 entry maps a (string) name to a function object, class object, or string.
 Function and class object docstrings found from ``M.__test__`` are searched, and
 strings are treated as if they were docstrings.  In output, a key ``K`` in
 ``M.__test__`` appears with name ::
 
    <name of M>.__test__.K
 
+An example is to place this block of code at the top of :file:`example.py`.
+
+.. code-block:: pycon
```

Comment:
`pycon` lexes this incorrectly, since the example inputs and outputs are actually in a multi-line string, but are displayed as literal Python console lines, while the surrounding Python code is displayed as Python output, greatly confusing me at first. Furthermore, when clicking the button to remove the prompts, the string content is kept while the surrounding code is removed. Changing this to `python` _should_ fix this (haven't tested this particular edge case, though).

```suggestion
.. code-block:: python
```